test/services/LibraryService.test.ts

```diff
@@ -42,7 +51,38 @@ describe('LibraryService', () => {
         });
     });
   });
+  describe('search', () => {
+    it('returns 200 and correct searchset bundle when query matches single resource', async () => {
+      await supertest(server.app)
+        .get('/4_0_1/Library')
+        .query({ url: 'http://example.com', status: 'active' })
+        .set('Accept', 'application/json+fhir')
+        .expect(200)
+        .then(response => {
+          expect(response.body.resourceType).toEqual('Bundle');
+          expect(response.body.total).toEqual(1);
+          expect(response.body.entry[0].resource).toEqual(LIBRARY_WITH_URL);
+        });
+    });
 
+    it('returns 200 and correct searchset bundle when query matches multiple resources', async () => {
+      await supertest(server.app)
+        .get('/4_0_1/Library')
+        .query({ status: 'active' })
+        .set('Accept', 'application/json+fhir')
+        .expect(200)
+        .then(response => {
+          expect(response.body.resourceType).toEqual('Bundle');
+          expect(response.body.total).toEqual(2);
+          expect(response.body.entry.find((e: fhir4.BundleEntry) => e.resource?.id === 'test')?.resource).toEqual(
```

Comment:
This is not a change request, but what do you think of this assertion style instead?

```ts
expect(response.body.entry).toEqual(
  expect.arrayContaining([
    expect.objectContaining<fhir4.BundleEntry>({
      resource: LIBRARY
    }),
    expect.objectContaining<fhir4.BundleEntry>({
      resource: LIBRARY_WITH_URL
    })
  ])
);
```

Totally cool if you prefer the .find, I'll leave it up to you